docs/resources/google_storage_bucket_objects.md

```diff
@@ -0,0 +1,75 @@
+---
+title: About the google_storage_bucket_objects Resource
+platform: gcp
+---
+
+# google\_storage\_bucket\_objects
+
+Use the `google_storage_bucket_objects` InSpec audit resource to test properties of a GCP storage bucket objects.
+
+<br>
+
+## Syntax
+
+A `google_storage_bucket_objects` resource block collects GCP bucket objects by project then tests that group.
+
+    describe google_storage_bucket_objects(bucket: 'bucket-name') do
+      it { should exist }
+    end
+
+Use this InSpec resource to enumerate IDs then test in-depth using `google_storage_bucket_object`.
+
+    google_storage_bucket_objects(bucket: 'bucket-name').object_names.each do |object_name|
+      describe google_storage_bucket_object(bucket: bucket_name, object: object_name) do
+        it { should exist }
+      end
+    end
+
+<br>
+
+## Examples
+
+The following examples show how to use this InSpec audit resource.
+
+### Test that there are no more than a specified number of storage buckets for the project
+
+    describe google_storage_bucket_objects(bucket: 'bucket-name') do
+      its('count') { should be <= 100}
```

Comment:
Very minor, but the linting of Ruby would state that there should be a space between 100 and the `}`